com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/handlers/v2/ModuleSorter.java

```diff
@@ -0,0 +1,25 @@
+package com.sap.cloud.lm.sl.mta.handlers.v2;
+
+import java.util.Comparator;
+
+import com.sap.cloud.lm.sl.mta.builders.v2.ModuleDependenciesCollector;
+import com.sap.cloud.lm.sl.mta.model.v1.DeploymentDescriptor;
+
+public class ModuleSorter extends com.sap.cloud.lm.sl.mta.handlers.v1.ModuleSorter {
```

Comment:
I think this class could be removed and to replace the inheritance with composition. So, make the ModuleSorter to accept ModuleDependenciesCollector and based on the version of the descriptor pass different ModuleDependenciesCollector. In this way you will eliminate the "empty" class here.